src/models/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";
+
```

Comment:
get rid of extra newlines

Reply:
lol that's autogenerated by amplify datastore. not sure if we need them but made the change